networking/networking.go

```diff
@@ -116,6 +125,54 @@ func Setup(podRoot string, podID types.UUID, fps []ForwardedPort, netList common
 	return &n, nil
 }
 
+// enableDefaultLocalnetRouting enables the route_localnet attribute on the supposedly default network interface.
+// This allows to setup a loopback NAT to access port forwardings to Pods on the host with the localhost address.
+func (n *Networking) enableDefaultLocalnetRouting() error {
+	routeLocalnetFormat := ""
+
+	defaultHostIP, err := n.GetDefaultHostIP()
+	if err != nil {
+		return err
+	}
+
+	defaultHostIPstring := defaultHostIP.String()
+	switch {
+	case strings.Contains(defaultHostIPstring, "."):
```

Comment:
Sure

Stefan Junker <notifications@github.com> schrieb am Fr., 29. Jan. 2016
15:37:

> In networking/networking.go
> <https://github.com/coreos/rkt/pull/2027#discussion_r51266774>:
>
> > @@ -116,6 +125,54 @@ func Setup(podRoot string, podID types.UUID, fps []ForwardedPort, netList common
> >  	return &n, nil
> >  }
> >
> > +// enableDefaultLocalnetRouting enables the route_localnet attribute on the supposedly default network interface.
> > +// This allows to setup a loopback NAT to access port forwardings to Pods on the host with the localhost address.
> > +func (n *Networking) enableDefaultLocalnetRouting() error {
> > +	routeLocalnetFormat := ""
> > +
> > +	defaultHostIP, err := n.GetDefaultHostIP()
> > +	if err != nil {
> > +		return err
> > +	}
> > +
> > +	defaultHostIPstring := defaultHostIP.String()
> > +	switch {
> > +	case strings.Contains(defaultHostIPstring, "."):
>
> I'd prefer to put it in an issue since we need a follow-up for this
> already. Are you okay with that?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/coreos/rkt/pull/2027/files#r51266774>.
>